.github/workflows/release.yml

```diff
@@ -21,18 +31,12 @@ jobs:
           token: ${{ secrets.ORG_GITHUB_TOKEN }}
       - name: Fetch All Tags
         run: git fetch --force --tags
-      - name: Get version bump
-        id: bump
-        env:
-          VERSION_BUMP: ${{ toJson(github.event.client_payload.bump) }}
-        run: |
-          echo version_bump=$(echo $VERSION_BUMP | tr -d "\"") >> $GITHUB_OUTPUT
       - name: Determine Next Version
         id: next_version
         uses: zwaldowski/semver-release-action@v4
         with:
           dry_run: true
-          bump: ${{ steps.bump.outputs.version_bump }}
+          bump: ${{ inputs.bump || toJson(github.event.client_payload.bump) }}
```

Comment:
This is the only part thats untested.  It worked for the `workflow_dispatch` but until its in main and we try with opslevel actions we won't know if this works.